packages/protoc-gen-connect-web/README.md

```diff
@@ -6,45 +6,93 @@ to think about serialization again.
 
 Learn more about Connect-Web at [github.com/bufbuild/connect-web](https://github.com/bufbuild/connect-web).
 
-`protoc-gen-connect-web` is a plugin for `protoc` and [`buf`](https://github.com/bufbuild/buf). 
-It generates services from your Protocol Buffer schema, and works in tandem with
-[@bufbuild/protoc-gen-es](https://www.npmjs.com/package/@bufbuild/protoc-gen-es), 
-the code generator plugin for all Protocol Buffer base types.
-
-
 ## Installation
 
-### With npm
+`protoc-gen-connect-web` is a code generator plugin for Protocol Buffer compilers, 
+like [buf](https://github.com/bufbuild/buf) and [protoc](https://github.com/protocolbuffers/protobuf/releases).
+It generates clients from your Protocol Buffer schema, and works in tandem with
+[@bufbuild/protoc-gen-es](https://www.npmjs.com/package/@bufbuild/protoc-gen-es),
+the code generator plugin for all Protocol Buffer base types. The code those two 
+plugins generate requires the runtime libraries [@bufbuild/connect-web](https://www.npmjs.com/package/@bufbuild/connect-web), 
+and [@bufbuild/protobuf](https://www.npmjs.com/package/@bufbuild/protobuf).
+
+To install the plugins and their runtime libraries, run:
 
 ```shell
-npm install @bufbuild/protoc-gen-es @bufbuild/protoc-gen-connect-web
+npm install --save-dev @bufbuild/protoc-gen-connect-web @bufbuild/protoc-gen-es
+npm install @bufbuild/connect-web @bufbuild/protobuf
 ```
 
-This will install the code generator plugins in `node_modules/.bin/protoc-gen-es`
-and `node_modules/.bin/protoc-gen-connect-web`.
-Note that npm does not add the executable to your `$PATH`. You can do so with:
+We use peer dependencies to ensure that code generator and runtime library are
+compatible with each other. Note that yarn and pnpm only emit a warning in this case.
```

Comment:
Hoping this might help a bit with the situation regarding https://github.com/bufbuild/connect-web/issues/239.